recipes/vim.rb

```diff
@@ -0,0 +1,104 @@
+#
```

Comment:
the recipe name should probably be "vim-sample.rb" to align with the other naming

Reply:
Done.